customizations.json

```diff
@@ -1,6 +1,15 @@
 {
   "featureGroupOverrides": {
-    "calls": {},
+    "calls": {
+      "GetGlobalPolicy": "Profiles",
+      "GetProfile": "Profiles",
+      "GetProfiles": "Profiles",
+      "GetTitlePlayersFromMasterPlayerAccountIds": "Profiles",
+      "SetGlobalPolicy": "Profiles",
+      "SetProfileLanguage": "Profiles",
+      "SetProfilePolicy": "Profiles",
+      "AdminSetPublisherData": "TitleDataManagement"

```

Comment:
out of curiosity how were these categorized previously?

Reply:
All these Profiles APIs are included in AccountManagement, and that Admin API is the one I was seeing in SharedGroups